src/include/migraphx/compile_options.hpp

```diff
@@ -32,9 +32,17 @@ inline namespace MIGRAPHX_INLINE_NS {
 
 struct compile_options
 {
-    bool offload_copy    = false;
+    /**
+     * Have MIGX allocate memory for parameters and add instructions
+     * to copy parameters and output to/from the GPU and host
```

Comment:
This should say `to copy parameters and output to/from an offload device like a GPU`. Since this is not a GPU-specific flag.